pom.xml

```diff
@@ -4,7 +4,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>revai</groupId>
     <artifactId>revai-java-sdk</artifactId>
-    <version>0.0.0</version>
+    <version>0.0.1</version>
```

Comment:
we will make it 1.0.0 before we release the first public version right?

Reply:
Yes this was just left over from me creating a jar for testing the ApiClient. Should I set it to 1.0.0 now?

Reply:
sure.